test_scripts/RC/AUDIO_LIGHT_HMI_SETTINGS/commonRCmodules.lua

```diff
@@ -446,6 +476,14 @@ function c.isSubscribed(pModuleType, pAppId)
   local rpc = "OnInteriorVehicleData"
   test.hmiConnection:SendNotification(c.getHMIEventName(rpc), c.getHMIResponseParams(rpc, pModuleType))
   mobSession:ExpectNotification(c.getAppEventName(rpc), c.getAppResponseParams(rpc, pModuleType))
+  :ValidIf(function(_,data)
+    if "AUDIO" == pModuleType and
+      nil ~= data.payload.moduleData.audioControlData.keepContext then
+      print( tostring(data.payload.moduleData.audioControlData.keepContext) )
```

Comment:
@GetmanetsIrina Guess print is useless here

Reply:
@dboltovskyi , please fid update in 8ce4954544173a2e887b1fe7278696ea6ae2d320